src/InterNations/Grid/GridManager.php

```diff
@@ -106,7 +107,9 @@ public function gridManagerResponse(array $parameters = array(), $view = null,
                 return $parameters;
             }
             else {
-                return $this->container->get('templating')->renderResponse($view, $parameters, $response);
+                $response = new Response();
```

Comment:
same here, i don't think we should be creating an empty response here, any changes to the response might be lost